vllm_spyre/v1/worker/spyre_worker.py

```diff
@@ -321,6 +322,18 @@ def _warmup_spyre_dynamic_size(self, special_token_ids):
         prompt_len = 42
         num_decode_tokens = 2
 
+        # Fix for batch size 1: set input batch to fit 2 requests for warmup
+        if model_runner.vllm_config.scheduler_config.max_num_seqs == 1:
+            model_runner.input_batch = InputBatch(
+                max_num_reqs=2,
```

Comment:
Why is it necessary to set the full input batch object instead of just setting `model_runner.input_batch.max_num_reqs=2`?